plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/AbstractMultiParentHook.java

```diff
@@ -103,7 +109,17 @@ private void cloneFromSCM(UpdateSite.Plugin currentPlugin, File parentPath, Stri
                 FileUtils.deleteDirectory(parentPath);
             }
             repository = scmManager.makeScmRepository(connectionURL);
-            CheckOutScmResult result = scmManager.checkOut(repository, new ScmFileSet(parentPath), new ScmTag(scmTag));
+            ScmProvider scmProvider = scmManager.getProviderByRepository(repository);
+            CheckOutScmResult result;
+            if (scmProvider instanceof GitExeScmProvider) {
+                CommandParameters parameters = new CommandParameters();
+                parameters.setString(CommandParameter.SHALLOW, TRUE.toString());
```

Comment:
:raised_eyebrow: 

```suggestion
                parameters.setString(CommandParameter.SHALLOW, "true");
```

suffices?

Reply:
why not but at the end it sounds similar 🤣